data/html/about.html

```diff
@@ -1,15 +1,31 @@
 %HEADER%
 
-    <div id="content-wrapper">
+    <script type="module">
+        if (window.appCreated === true) {
+            vueApp.mount('#content');
+        }
+        else {
+            window.addEventListener('appCreated', () => {
+                vueApp.mount('#content');
+            });
+        }
+    </script>
```

Comment:
The repetition can be prevented like this:
```suggestion
    <script type="module">
        (window.appCreated ? Promise.resolve() : new Promise(resolve => window.addEventListener('appCreated', resolve)))
            .then(() => vueApp.mount('#content'));
    </script>
```